packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,223 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized, useLocalization } from '@fluent/react';
+import React, { useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from '@reach/router';
+import { HomePath } from '../../constants';
+import {
+  logViewEvent,
+  settingsViewName,
+  usePageViewEvent,
+} from '../../lib/metrics';
+import { useAccount, useAlertBar } from '../../models';
+import { InputText } from '../../components/Settings/InputText';
+import RememberPassword from '../../components/App/RememberPassword';
+
+// --forceEmail-- is a hint to the signup page that the user should not
```

Comment:
I honestly can not remember when/where we use the force email feature of reset password. I think it could help simplify our code if we didn't have this extra check.

@LZoog Thoughts on not having it?

Reply:
I was thinking it might be related to entering the flow from a relying party, though I'm really not sure and haven't been able to trigger this flow. If forceEmail is received, the "remember password" link redirects to force_auth instead of the regular signin page.